RemoteDocumentLoader.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace ML\JsonLD;
+
+use ML\JsonLD\Exception\JsonLdException;
+
+interface RemoteDocumentLoader
```

Comment:
The other interfaces in this project all have an Interface suffix in their name, so please rename this to `DocumentLoaderInterface`